core/src/main/java/hudson/PluginManager.java

```diff
@@ -446,6 +478,40 @@ public void run(Reactor reactor) throws Exception {
         }});
     }
 
+    @Extension
+    public final static PluginManagerAdministrativeMonitor NOTICE = new PluginManagerAdministrativeMonitor();
```

Comment:
@daniel-beck 

> No reason to have a public static field for this.

+1

> To add, you could look up the extension instance from the extension list.

I tried yesterday and it doesn't work - the extension point is null at that point. I guess that because they are not loaded at this point.